test/express-rate-limit-test.js

```diff
@@ -425,6 +425,39 @@ describe("express-rate-limit node module", () => {
     assert(!store.decrement_was_called, "decrement was called on the store");
   });
 
+  it("should decrement hits with success response with custom 'requestWasSuccessful' option", async () => {
+    const store = new MockStore();
+    createAppWith(
+      rateLimit({
+        skipSuccessfulRequests: true,
+        requestWasSuccessful: function (req, res) {
+          return res.statusCode === 200;
+        },
```

Comment:
I'm a little iffy about this because, for this test, the behavior is the same as the default `requestWasSuccessful` function.

What if you made it work off of the querystring? e.g. `return req.query.success === "1";` here and in the next test, and then do`.get("/?success=1").` here and `.get("/?success=0")` in the next test?